Watch Extension/FeedingView.swift

```diff
@@ -35,11 +44,7 @@ struct FeedingView: View {
                             .default(Text("Stop")) {
                                 self.store.send(.stop(self.feeding))
                             },
-                            // TODO: make this real, which can probably be accomplished
-                            // by having a pullback on the store's state mutation
-                            .default(Text("Pause/Resume")) {
-                                self.store.send(.pause(self.feeding))
-                            },
+                            actionSheetButton(self.feeding)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/880c027cb3a63d59b8015ec9e1cab691a35d0496/Watch Extension/FeedingView.swift#L47"></span>Multi-line collection literals should have trailing commas.<br />
<code>trailing_comma</code> <code>FeedingView.swift:47</code></td>
    </tr>
  </tbody>
</table>